ci-client.sh

```diff
@@ -0,0 +1,2 @@
+npm install
+npm run test -- --reporter xunit --file tmp/reports/client-test-sprockets-result.xml
```

Comment:
mocha command does not have `--file` option.
(Do you misunderstand [mocha-phantomjs](https://github.com/nathanboktae/mocha-phantomjs#usage) options?)

It seems good to use `--reporter-options output=filename.xml` instead.
https://mochajs.org/#undocumented-reporters